qcfractal/queue/parsl_adapter.py

```diff
@@ -76,6 +76,14 @@ def submit_tasks(self, tasks: Dict[str, Any]) -> List[str]:
 
             # Form run tuple
             func = self.get_app(spec["spec"]["function"])
+            # Trap QCEngine Memory and CPU
```

Comment:
This looks like a good candidate for the BaseAdapter to have something like `kwargs = self.message_kwargs(spec["spec"])` or similar so we do not repeat this many times.